src/it/WriteActiveProfilePropertiesToOutputFile/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>@project.groupId@</groupId>
+		<artifactId>it-test-parent</artifactId>
+		<version>@project.version@</version>
+		<relativePath>../it-test-parent/pom.xml</relativePath>
+	</parent>
+	<!-- Build description -->
+	<artifactId>WriteActiveProfilePropertiesToOutputFile</artifactId>
+	<packaging>pom</packaging>
+	<name>${project.artifactId}</name>
+
+	<!-- Build plugins and extensions -->
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>@project.groupId@</groupId>
+				<artifactId>properties-maven-plugin</artifactId>
+				<version>${testing.versionUnderTest}</version>
```

Comment:
This can simply be handled by using `@project.version@`. There is no need for a supplemental property here in the integration test.